springboard/config.py

```diff
@@ -4,4 +4,5 @@ def includeme(config):
     config.add_route('category', '/category/{uuid}/')
     config.add_route('page', '/page/{uuid}/')
     config.add_route('flat_page', '/{slug}/')
+    config.add_route('api_notify', '/api/notify/', request_method='POST')
```

Comment:
It hasn't, but I want to home it in the `unicore.distribute` package as a standalone capable thing.